README.md

```diff
@@ -215,84 +215,221 @@ result = mq.index("my-first-index").search('adventure', searchable_attributes=['
 
 ```
 
-### Delete documents
+### Multi modal and cross modal search
 
-Delete documents.
+To power image and text search, Marqo allows users to plug and play with CLIP models from HuggingFace. **Note that if you do not configure multi modal search, image urls will be treated as strings.** To start indexing and searching with images, first create an index with a CLIP configuration, as below:
 
 ```python
 
-results = mq.index("my-first-index").delete_documents(ids=["article_591", "article_602"])
+settings = {
+    "treat_urls_and_pointers_as_images":True,   # allows us to find an image file and index it 
+    "model":"ViT-L/14"
+}
+response = mq.create_index("my-multimodal-index", **settings)
 
 ```
 
-### Delete index
-
-Delete an index.
+Images can then be added within documents as follows. You can use urls from the internet (for example S3) or from the disk of the machine:
 
 ```python
 
-results = mq.index("my-first-index").delete()
+response = mq.index("my-multimodal-index").add_documents([{
+    "My Image": "https://upload.wikimedia.org/wikipedia/commons/thumb/b/b3/Hipop%C3%B3tamo_%28Hippopotamus_amphibius%29%2C_parque_nacional_de_Chobe%2C_Botsuana%2C_2018-07-28%2C_DD_82.jpg/640px-Hipop%C3%B3tamo_%28Hippopotamus_amphibius%29%2C_parque_nacional_de_Chobe%2C_Botsuana%2C_2018-07-28%2C_DD_82.jpg",
```

Comment:
Wikipedia images, like this, aren't very stable in my experience. We have these Hippo images that are quite useful:
- realistic: `https://raw.githubusercontent.com/marqo-ai/marqo-api-tests/mainline/assets/ai_hippo_realistic.png`
- artefact: `https://raw.githubusercontent.com/marqo-ai/marqo-api-tests/mainline/assets/ai_hippo_statue.png`

Reply:
That section is a copy-paste from the current README, I will test and update it with one of the more stable links though as I also agree that they would be better.

Reply:
@pandu-k I have added a commit to use the image links you provided